src/app/presentation/components/user-invite/user-invite.component.html

```diff
@@ -22,10 +22,10 @@
               to<span class="toUserColor"> {{item.toUserName}}</span>
            </div>
           </div>
-          <div>
+          <div *ngIf="(item.fromUserName == currentUserName && currentUserRole.role == 'Manager') || currentUserRole.role == 'Owner'">
```

Comment:
Please rewrite it with out ngif on view

Reply:
https://sharing.clickup.com/t/h/1tf90az/0S9B98ADVGG2OUG